### PR TITLE
Rando commit

### PR DESCRIPTION
**Pull request by testbot2100910**
Make things const with this commit

### CODE REVIEW

File: ramda.js [hunk 1]
```diff
@@ -33,7 +33,7 @@
      *      var greet = R.replace('{name}', R.__, 'Hello, {name}!');
      *      greet('Alice'); //=> 'Hello, Alice!'
      */
-    var __ = { '@@functional/placeholder': true };
+    const __ = { '@@functional/placeholder': true };
```
Comment (shantanuraj) [inline ramda.js]: Should use let here

Comment (testbot2100910) [inline ramda.js]: Hey

File: ramda.js [hunk 2]
```diff
@@ -33,7 +33,7 @@
...
 
     /* eslint-disable no-unused-vars */
     var _arity = function _arity(n, fn) {
```
Comment (shantanuraj) [inline ramda.js]: Fix this

File: ramda.js [hunk 3]
```diff
@@ -33,7 +33,7 @@
      *      var greet = R.replace('{name}', R.__, 'Hello, {name}!');
      *      greet('Alice'); //=> 'Hello, Alice!'
      */
-    var __ = { '@@functional/placeholder': true };
```
Comment (shantanuraj) [inline ramda.js]: Use let here.

Comment (shantanuraj) [inline ramda.js]: Heyo

Comment (shantanuraj) [inline ramda.js]: Heyo 2

Comment (shantanuraj) [inline ramda.js]: Heyo 3

Comment (shantanuraj) [inline ramda.js]: A

Comment (shantanuraj) [inline ramda.js]: m

Comment (shantanuraj) [inline ramda.js]: m

Comment (shantanuraj) [inline ramda.js]: m4

Comment (shantanuraj) [inline ramda.js]: m5

File: ramda.js [hunk 4]
```diff
@@ -33,10 +33,10 @@
      *      var greet = R.replace('{name}', R.__, 'Hello, {name}!');
      *      greet('Alice'); //=> 'Hello, Alice!'
```
Comment (shantanuraj) [inline ramda.js]: Doesn't look like anything to me

File: ramda.js [hunk 5]
```diff
@@ -33,10 +33,10 @@
...
      */
-    var __ = { '@@functional/placeholder': true };
+    let __ = { '@@functional/placeholder': true }
 
     /* eslint-disable no-unused-vars */
-    var _arity = function _arity(n, fn) {
+    let _arity = function _arity(n, fn) {
         /* eslint-disable no-unused-vars */
         switch (n) {
```
Comment (shantanuraj) [inline ramda.js]: Doesn't look like anything to me

Comment (shantanuraj) [inline ramda.js]: Why not?

File: ramda.js (hunk 3 above)
Comment (shantanuraj) [inline ramda.js]: This one is a reject.

Comment (shantanuraj) [inline ramda.js]: Now to accept

Comment (shantanuraj) [inline ramda.js]: Really accept

Comment (shantanuraj) [inline ramda.js]: Accept it

Comment (shantanuraj) [inline ramda.js]: Accept forever

Comment (shantanuraj) [inline ramda.js]: Accept this time

File: ramda.js (hunk 5 above)
Comment (shantanuraj) [inline ramda.js]: Accept

File: ramda.js (hunk 3 above)
Comment (shantanuraj) [inline ramda.js]: Hey yo

Comment (shantanuraj) [inline ramda.js]: Fog

File: ramda.js (hunk 4 above)
Comment (shantanuraj) [inline ramda.js]: Greeting Alice

File: ramda.js (hunk 3 above)
Comment (shantanuraj) [inline ramda.js]: Hey

File: ramda.js (hunk 5 above)
Comment (shantanuraj) [inline ramda.js]: 👍 Accept

Comment (shantanuraj) [inline ramda.js]: Accepted!

Comment (shantanuraj) [inline ramda.js]: 👍 Accept

File: ramda.js (hunk 3 above)
Comment (shantanuraj) [inline ramda.js]: Accept

File: ramda.js [hunk 6]
```diff
@@ -33,10 +33,10 @@
      *      var greet = R.replace('{name}', R.__, 'Hello, {name}!');
```
Comment (shantanuraj) [inline ramda.js]: Hello

Comment (shantanuraj) [inline ramda.js]: Accept it

Comment (shantanuraj) [inline ramda.js]: accept

Comment (shantanuraj) [inline ramda.js]: patched

Comment (shantanuraj) [inline ramda.js]: patched

Comment (shantanuraj) [inline ramda.js]: patched

Comment (shantanuraj) [inline ramda.js]: patched

Comment (shantanuraj) [inline ramda.js]: patched

Comment (shantanuraj) [inline ramda.js]: reject

Comment (shantanuraj) [inline ramda.js]: wontfix

Comment (shantanuraj) [inline ramda.js]: patched

Comment (shantanuraj) [inline ramda.js]: patched

Comment (shantanuraj) [inline ramda.js]: dasdasda

Comment (shantanuraj) [inline ramda.js]: Patched

Comment (shantanuraj) [inline ramda.js]: wontfix

Comment (shantanuraj) [inline ramda.js]: patched

Comment (shantanuraj) [inline ramda.js]: wontfix

Comment (shantanuraj) [inline ramda.js]: patched

File: ramda.js [hunk 7]
```diff
@@ -33,10 +33,10 @@
...
      *      greet('Alice'); //=> 'Hello, Alice!'
      */
-    var __ = { '@@functional/placeholder': true };
+    let __ = { '@@functional/placeholder': true }
 
     /* eslint-disable no-unused-vars */
```
Comment (shantanuraj) [inline ramda.js]: Straight up fire

Comment (shantanuraj) [inline ramda.js]: Accept

Comment (shantanuraj) [inline ramda.js]: wontfix

Comment (shantanuraj) [inline ramda.js]: wontfix

Comment (shantanuraj) [inline ramda.js]: reject

Comment (shantanuraj) [inline ramda.js]: This is a test

Comment (shantanuraj) [inline ramda.js]: This is a test too

Comment (shantanuraj) [inline ramda.js]: Is it?

Comment (shantanuraj) [inline ramda.js]: This is a test

Comment (shantanuraj) [inline ramda.js]: Welcome to a world of tests

Comment (shantanuraj) [inline ramda.js]: In a world of commits

Comment (shantanuraj) [inline ramda.js]: one man must fight the world to test some changes

Comment (shantanuraj) [inline ramda.js]: Reject this

Comment (shantanuraj) [inline ramda.js]: Reject this too

Comment (shantanuraj) [inline ramda.js]: Accept this

Comment (shantanuraj) [inline ramda.js]: 👍

Comment (shantanuraj) [inline ramda.js]: accept

Comment (shantanuraj) [inline ramda.js]: 👍

File: ramda.js [hunk 8]
```diff
@@ -33,10 +33,10 @@
...
-    var _arity = function _arity(n, fn) {
+    let _arity = function _arity(n, fn) {
```
Comment (shantanuraj) [inline ramda.js]: Make functional

File: ramda.js [hunk 9]
```diff
@@ -33,10 +33,10 @@
      *      var greet = R.replace('{name}', R.__, 'Hello, {name}!');
      *      greet('Alice'); //=> 'Hello, Alice!'
      */
-    var __ = { '@@functional/placeholder': true };
+    let __ = { '@@functional/placeholder': true }
```
Comment (shantanuraj) [inline ramda.js]: Take less time next time

Comment (shantanuraj) [inline ramda.js]: 🔔 Patched address in some commit

Comment (shantanuraj) [inline ramda.js]: 👍 Accept

**Pull request merged**